stl/inc/fstream

```diff
@@ -281,6 +284,7 @@ public:
     }
 
     basic_filebuf* open(const char* _Filename, ios_base::openmode _Mode, int _Prot = ios_base::_Default_open_prot) {
+        this->_Binary = _Mode & ios::binary;

```

Comment:
We use `this->` only when it is required to find template base member. Occurs below.